guides/customizing-invoices.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc
+sublayout: guide
+title: Customizing Invoices
+---
+
+Invoices provided by Quay.io try to provide as much information as possible.
+However, it be necessary to include extra information for regulations such as Value Added Tax (VAT).
+In order to solve this problem, Quay.io supports the ability to automatically customize invoices by adding custom fields and their contents.
+
+### Adding new data
+
+Navigate to the _Billing Invoices_ tab of your organization (`https://quay.io/organization/YOURORG?tab=invoices`).
+In the top-right corner of the page, select the drop-down as pictured below.
+Select "Add Custom Invoice Field" and fill in the subsequent modal dialog with the name of the field and its value.
+Now selecting the download button on an invoice will produce an invoice containing the new custom information.
+
+<div class="article-image">
+  <a href="add-custom-invoice-field.png"><img src="add-custom-invoice-field.png" style="width:75%;"></a>
+</div>
+
+
+### Removing data
+
+Navigate to the _Billing Invoices_ tab of your organization (`https://quay.io/organization/YOURORG?tab=invoices`).
+In the top-right corner of the page, select the drop-down as pictured below.
+Select the trash can icon next to the custom field that should be removed.
```

Comment:
I think people understand what a Click is on a mobile device too, but I don't feel strongly about this one way or the other